website/frontend_types.py

```diff
@@ -101,4 +102,5 @@ def from_teacher_adventure_database_row(row):
             save_name=row['name'],
             start_code='',  # Teacher adventures don't seem to have this
```

Comment:
True! These start codes in general need a bit more thinking! I will open an issue about that later!